BootloaderCommonPkg/Library/TpmLib/TpmLib.c

```diff
@@ -806,15 +838,18 @@ TpmChangePlatformAuth (
   randomize platform auth, add EV_SEPARATOR event etc) as indicated in
   PC Client Specific Firmware Profile specification.
 
+  @param[in] FwDebugEnabled    Firmware Debug Mode enabled/disabled.
+
   @retval RETURN_SUCCESS   Operation completed successfully.
   @retval Others           Unable to finish handling ReadyToBoot events.
 **/
 RETURN_STATUS
 TpmIndicateReadyToBoot (
-  VOID
+  IN UINT8 FwDebugEnabled
   )
 {
-  if ( (TpmExtendSecureBootPolicy () != EFI_SUCCESS) ||
+  if ( (MeasureLaunchOfFirmwareDebugger (FwDebugEnabled) != EFI_SUCCESS) ||

```

Comment:
considering breaking it down into multiple statements to make it more readable.

Reply:
Done